spec/models/note_spec.rb

```diff
@@ -14,4 +14,116 @@
   it 'has a valid factory' do
     expect(subject).to be_valid
   end
+
+  describe '.content_length' do
+    let(:north_utility) { create(:north_utility, code: 1) }
+    let(:south_utility) { create(:south_utility, code: 2) }
+
+    let(:north_user) { create(:user, utility_id: north_utility.id) }
+    let(:south_user) { create(:user, utility_id: south_utility.id) }
+
+    context 'when North Utility' do
+      let(:north_note) { create(:note, user_id: north_user.id, content: content, note_type: :critique) }
+
+      context 'with content <= 50 words' do
+        let(:content) { Faker::Lorem.words(number: 50).join(' ') }
+
+        it 'return "short"' do
+          expect(north_note.content_length).to eq('short')
+        end
+      end
```

Comment:
Y acá adentro creamos nuestra utility (que es lo que cambia entre tus distintos contextos)

```suggestion
      let(:utility) { create(:north_utility) }

      context 'with content <= 50 words' do
        let(:content) { Faker::Lorem.words(number: 50).join(' ') }

        it 'return "short"' do
          expect(note.content_length).to eq('short')
        end
      end
```

Esto a futuro nos permitiría reutilizar tests por ejemplo, porque nos abstraemos de quién es la utility, nos importa simplemente la validación que ejecuta la nota 🤗 

Lo mismo aplica a todos los tests siguientes y a la `south_utility` 👇🏻